embedx/config.schema.json

```diff
@@ -1128,6 +1128,18 @@
                       "description": "If set to false the password validation fails when the network or the Have I Been Pwnd API is down.",
                       "type": "boolean",
                       "default": true
+                    },
+                    "min_password_length": {
+                      "title": "Minimum Password Length",
+                      "description": "Defines the minimum length of the password.",
+                      "type": "integer",
+                      "default": 6
```

Comment:
Please make the default `8`. We changed the default to be 8 in #2009, and it had quite some implications especially for automated tests and the login UI, so I would like to keep that default value.